hyrisecockpit/workload_generator/workloads/tpch_0_1/08.sql

```diff
@@ -1,11 +1,11 @@
 SELECT o_year, SUM(CASE
-																							WHEN nation = 'MOZAMBIQUE' THEN volume
+																							WHEN nation_tpch_0_1 = 'MOZAMBIQUE' THEN volume
 																							ELSE 0
 																			END) / SUM(volume) AS mkt_share
 FROM
-		(SELECT SUBSTR(o_orderdate, 1, 4) AS o_year, l_extendedprice * (1 - l_discount) AS volume, n2.n_name AS nation
-			FROM part, supplier, lineitem, orders, customer, nation n1, nation n2, region
-			WHERE p_partkey = l_partkey
+		(SELECT SUBSTR(o_orderdate, 1, 4) AS o_year, l_extendedprice * (1 - l_discount) AS volume, n2.n_name AS nation_tpch_0_1
```

Comment:
This should not be renamed:

```suggestion
		(SELECT SUBSTR(o_orderdate, 1, 4) AS o_year, l_extendedprice * (1 - l_discount) AS volume, n2.n_name AS nation
```